packages/flutter_driver/lib/src/extension.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:developer';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/src/instrumentation.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:flutter_driver/src/find.dart';
+import 'package:flutter_driver/src/gesture.dart';
+import 'package:flutter_driver/src/health.dart';
+import 'package:flutter_driver/src/message.dart';
+
+const String _extensionMethod = 'ext.flutter_driver';
+
+bool _flutterDriverExtensionEnabled = false;
+
+/// Enables Flutter Driver VM service extension.
+///
+/// This extension is required for tests that use `package:flutter_driver` to
+/// drive applications from a separate process.
+///
+/// Call this function prior to running your application, e.g. before you call
+/// `runApp`.
+void enableFlutterDriverExtension() {
+  if (_flutterDriverExtensionEnabled) return;
+  FlutterDriverExtension extension = new FlutterDriverExtension();
+  registerExtension(_extensionMethod, (_, Map<String, String> params) {
```

Comment:
in general, avoid using _ for argument names, so that readers can see what the argument was